### ISSUE TITLE
Make a moving element invisible outside of its parent container

### ISSUE THREAD

**Issue by interk0t** [isaacHagoel/svelte-dnd-action#516]
Hello!
I ran into the following problem.
When moving an object, as you can see in the screenshot, it is visible outside the parent container, I also noticed that the moved object (element) itself is not created in the same hierarchy with the rest of the elements of my component, but in the <body> tag, with the position "fixed" .
Is there any way to make an element is not visible outside the parent container that has the "hidden" style applied? Thank you

![Screenshot_1](https://github.com/isaacHagoel/svelte-dnd-action/assets/110355982/d5133bb5-66a3-4437-b6f5-45c306183982)

**Comment by isaacHagoel** [comment]
do u want it to be auto dropped when it leaves the container or just disappear? 
if the latter how should the user know they are still dragging it?

**Comment by interk0t** [comment]
> do u want it to be auto dropped when it leaves the container or just disappear? if the latter how should the user know they are still dragging it?

no, I don’t want it to be deleted, I just want it not to be visible if the user moves it outside the parent container, as in any other case, when I don’t want the content to be displayed outside of any container, I set the property hidden to the parent and then children is not visible if it goes beyond its limits. 
If you look at the screenshot, the parent is the blue container and when I move the icon outside of it, it is still visible...

**Comment by isaacHagoel** [comment]
Closing because there is not enough current reproducible information to take action and this report predates substantial changes now present in v0.9.77. If the problem still occurs, please open a new focused issue with browser, Svelte and library versions plus a minimal reproduction.
